airbyte-webapp/src/pages/SettingsPage/pages/ConnectorsPage/components/CreateConnector.tsx

```diff
@@ -77,10 +82,30 @@ const CreateConnector: React.FC<IProps> = ({ type }) => {
 
   return (
     <>
-      {type === "configuration" ? null : (
```

Comment:
As far as I could tell, `"configuration"` is no longer a value passed into the `type` prop, so I removed that check here